### ISSUE TITLE
Immersive Reader and Accessibility Tools (JAWS, ZoomText)

### ISSUE THREAD

**Issue by MNL79** [OfficeDev/microsoft-teams-apps-company-communicator#1137]
I've deployed v5.3 to our tenant and we are almost ready to fully start using it with our 22,000+ users.

However, our Accessibility Team have ran tests and it has failed the app. JAWS, ZoomText and Texthelp fail to read the message, it sees the card and will only read out the time stamp - but not the actual message in the card.

Searching through the closed issues, I can see advice to upgrade to 5.4.1 - however, this does not seem to fix the issue, according to other community members:

https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/issues/901
https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/issues/934

Before I go through the full effort of upgrading to the latest version - please can you confirm it is definitely reading out the message in the cards posted by the app?

I cannot deploy this app, if it is not accessible for our main AT apps. I've noticed in the Teams App, Decisions, it also posts cards and these are not readable by AT screen readers or Immersive Reader either.

**Comment by gsv022** [comment]
Hi @MNL79 ,
Thanks for raising the query. 

Company Communicator v5.4.1 complies with Microsoft accessibility standards and guidelines for Web, Windows and Mac OS (Grade C). It has not been tested for immersive reader and supports for only JAWS and NVDA for Web and Desktop versions. 

Please go ahead with the latest version. Keep us posted.

**Comment by MNL79** [comment]
So, you can confirm that JAWS will read out what is in the card from any message sent by Communicator?

**Comment by gsv022** [comment]
Yes, please go ahead.
